README.md

```diff
@@ -1,30 +1,33 @@
-# Birkhoff's ergodic theorem in LEAN4
+# Birkhoff's ergodic theorem in Lean 4
 
 ![Lean build](https://github.com/mseri/birkhoff/actions/workflows/build.yml/badge.svg)
 
-This repository contains the output of our project
-at the [Machine-Checked Mathematics Workshop](https://www.lorentzcenter.nl/machine-checked-mathematics.html)
-at the Lorentz Center, 10-14 July 2023.
+This project was initiated at the [Machine-Checked Mathematics Workshop](https://www.lorentzcenter.nl/machine-checked-mathematics.html) at the Lorentz Center, 10-14 July 2023.
 
-The project was developed with @marcolenci and Guillaume Dubach,
-under the support and supervision of Sébastien Gouëzel.
+Developed with @mseri, @marcolenci and Guillaume Dubach, under the support and supervision of Sébastien Gouëzel.
```

Comment:
I'm easy about this. We could leave it for the moment. I'll focus on adding more of the maths soon! 

Following the mathlib style the list of authors should be within the individual files.

At some point in the future it would be good that someone who comes across the repo knows that the there of us are actively developing/maintaing it and so knows who to contact.

Reply:
Ok!